package.json

```diff
@@ -20,7 +20,7 @@
     "planets"
   ],
   "author": "Hemantkumar Goswami",
-  "license": "GPL-3.0",
+  "license": "GPL-2.0",
```

Comment:
Why would you want to change the version of GPL?

Reply:
it was not my intention to _change_ the license type, only to make all references _consistent_. The license type is referenced in 3 different files: `README.md`, `package.json`, and the `LICENSE` file itself. The README and the LICENSE both say GPL 2.0 but `package.json` says GPL 3.0. Also, I can see in the commit history that you "downgraded" the license type from GPL 3.0 to GPL 2.0 on 03/05/2018, so I presumed GPL 2.0 is the desired license type. Therefore I changed `package.json` to match what is already in `LICENSE` and the `README.md`

Reply:
I see, its my bad then, do you want to make all of it 3.0 please?